Makefile

```diff
@@ -653,3 +630,9 @@ test-xcode-watchOS-release:
 		-scheme SwiftProtobuf_watchOS \
 		-configuration Release \
 		build $(XCODEBUILD_EXTRAS)
+test-conformance: build check-for-protobuf-checkout Sources/Conformance/failure_list_swift.txt Sources/Conformance/text_format_failure_list_swift.txt
```

Comment:
Make my want a blank line between these two targets

Reply:
Yep it wants that!